bytestring_splitter/__init__.py

```diff
@@ -229,19 +254,20 @@ def repeat(self, splittable, as_set=False):
 class BytestringKwargifier(BytestringSplitter):
     processed_objects_container = dict
 
-    def __init__(self, receiver=None, **kwargs):
-        self.receiver = receiver
-        BytestringSplitter.__init__(self, *kwargs.items())
+    def __init__(self, _receiver=None, _additional_kwargs=None, **parameter_pairs):
+        self.receiver = _receiver
+        self._additional_kwargs = _additional_kwargs or {}
+        BytestringSplitter.__init__(self, *parameter_pairs.items())
```

Comment:
What - you don't like `super`?

Reply:
That's right.  Haha, I dunno why I wrote that.  I got nothin'.

Reply:
But... think of the children!